_includes/analytics.html

```diff
@@ -1,9 +1,3 @@
-<!-- Global site tag (gtag.js) - Google Analytics -->
```

Comment:
This was one case where it was not safe to do `theirs`. Keep the previous version of this file.